Cannot read property 'add' of undefined on workbox-webpack-plugin v5

**Library Affected**:
workbox-webpack-plugin v5.0.0-alpha.0 & 1 & 2

**Browser & Platform**:
all browsers

**Issue or Feature Request Description**:
When building I get the error:
```
 ERROR  Failed to compile with 1 errors2:48:35 PM

Cannot read property 'add' of undefined

 ERROR  Build failed with errors.
```
This only happens when I include any of the workbox plugins
I'm using Vue CLI but not the PWA plugin because I need some of the features/fixes from v5.0.0-alpha.2

```
> Executing task: npm run build:web <


> anchor-chat@0.1.0 build:web /home/lukas2005/Dokumenty/Projects/Anchor/web
> vue-cli-service build --mode production.web

starting webConfig

⠙  Building for production.web...clean-webpack-plugin: /home/lukas2005/Dokumenty/Projects/Anchor/web/dist/**/* has been removed.
Starting type checking and linting service...
Using 1 worker with 2048MB memory limit
⠇  Building for production.web...

 ERROR  Failed to compile with 1 errors                                                                                                                                            10:30:56 PM

 error  

Cannot read property 'add' of undefined

 ERROR  Build failed with errors.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! anchor-chat@0.1.0 build:web: `vue-cli-service build --mode production.web`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the anchor-chat@0.1.0 build:web script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/lukas2005/.npm/_logs/2019-08-21T20_30_57_100Z-debug.log
The terminal process terminated with exit code: 1
```
I'm using Vue CLI 3.10.0 which appears to be using Webpack 4.28.4

--- a/packages/workbox-webpack-plugin/src/lib/get-manifest-entries-from-compilation.js
+++ b/packages/workbox-webpack-plugin/src/lib/get-manifest-entries-from-compilation.js
@@ -66,7 +66,10 @@ function assetToChunkNameMapping(stats) {
 
   for (const [chunkName, {assets}] of Object.entries(stats.namedChunkGroups)) {
     for (const assetName of assets) {
-      mapping[assetName].add(chunkName);
+      // See https://github.com/GoogleChrome/workbox/issues/2194
+      if (mapping[assetName]) {
+        mapping[assetName].add(chunkName);
+      }
     }
   }
 

